src/user-icon/user-icon.scss

```diff
@@ -92,8 +92,8 @@
 
     &::after {
       content: '';
-      width: calc(100% / 1.5);
-      height: calc(100% / 1.5);
+      width: $iui-s;
+      height: $iui-s;
```

Comment:
I don't like that all icons are the same size.
![image](https://user-images.githubusercontent.com/83585998/159878811-18887419-1b0b-4605-9088-efdac0ac8204.png)

Though, I'm not sure how to solve this, because if we changed sizes based on user icon size, it would like 1px change and we don't have variables for that. So maybe it's ok.

Reply:
We have 3 sizes of user icons. Cant we just specify for each?